src/args.rs

```diff
@@ -41,6 +45,10 @@ pub fn get_cli_args() -> Args {
         .author(env!("CARGO_PKG_AUTHORS"))
         .about(env!("CARGO_PKG_DESCRIPTION"))
         .setting(AppSettings::SubcommandRequiredElseHelp)
+        .subcommand(
+            SubCommand::with_name("configure-network")
```

Comment:
Can we choose a more precise name like `sync-connections`? Then if we want to have a more encompassing command for doing different things on boot time then we can introduce additional command like `boot-time-configure` or similar and add a list of specific sub-commands to `os-config.json` that can be executed one after another. That could include `generate-api-key` and the new command introduced here. This will be easier for integration testing the specific sub-commands.